frontend/src/metabase/qb/components/actions/XRaySegment.jsx

```diff
@@ -0,0 +1,24 @@
+/* @flow */
+
+import type {
+    ClickAction,
+    ClickActionProps
+} from "metabase/meta/types/Visualization";
+
+export default ({ card, tableMetadata }: ClickActionProps): ClickAction[] => {
+    console.log(card);
+    if (card.id) {
+        return [
+            {
+                name: "underlying-data",
+                title: "XRay this Card",
+                icon: "table",
+                url: () => {
+                    return "/xray/card/" + card.id;
```

Comment:
@kdoh do we have a card ID route/component? 

This action allows you to take something in segment view (eg, a table or a segment) and go to an X-Ray. This should only ever be arrived at for a "table" card, so perhaps a simple copy+pasta of the table x-ray might suffice here? 

Does that feel within scope to you?

Also, @sbelak are we doing anything special in the card x-rays to determine whether something is eg a timeseries and do something interesting with it?